pages/test_analytics/ci_environments.md.erb

```diff
@@ -0,0 +1,55 @@
+# CI Environments
+
+Buildkite Test Analytics collectors automatically detect common continuous integration (CI) environments.
+If available, test collectors gather information about your test runs, such as branch names and build IDs.
+Test collectors gather information from the following CI environments:
+
+- [Buildkite](#buildkite)
+- [CircleCI](#circleci)
+- [GitHub Actions](#github-actions)
+
+If you use a test collector in any other environment, the test run will be assigned a unique identifier and attributed to a "generic" CI system.
+Other information, such as branch names or commit hashes, will not be collected.
```

Comment:
If we want to emphasize this piece a little more, we could move this to a section — "Using other CI systems" perhaps — so that it's less "only these a supported" and more "we make it easy for these, but it's possible with others."

Reply:
Here I'd be tempted to move the Others into a heading, use the built in TOC only, and leave the intro text more generic. What do you reckon?

Reply:
Yes, for sure! Marketing has been emphasizing that Test Analytics will work with any CI. Should find a home for the generic version of CI, and your suggestion to put this is a new section called Other makes sense to me.

`If you're using other CI providers, you need to configure these environment variables yourself. The only required environment variable is the unique key for the build that initiated the Test Analytics run (BUILDKITE_ANALYTICS_KEY), but some features of Test Analytics aren't available if the other environment variables are not set:`

```
Variable | Description and source
-- | --
BUILDKITE_ANALYTICS_KEY | a unique key for the build that initiated the Test Analytics run
BUILDKITE_ANALYTICS_URL | URL that links to the build
BUILDKITE_ANALYTICS_BRANCH | the branch or reference that this run is for
BUILDKITE_ANALYTICS_SHA | the commit SHA for the head of the branch
BUILDKITE_ANALYTICS_NUMBER | the build number of the build
BUILDKITE_ANALYTICS_JOB_ID | the id of a job within the build
BUILDKITE_ANANLYTICS_MESSAGE | the commit message for the head of the branch
```

Reply:
OK, I can work with this.